infretis/classes/engines/gromacs.py

```diff
@@ -674,12 +605,12 @@ def _propagate_from(
                 # Also provide the loaded positions since they are
                 # available:
                 system.pos = data["x"]
-                system.vel = data.get("v", None)
+                system.vel = data["v"]
                 if system.vel is not None and reverse:
                     system.vel *= -1
                 # ##### length = box_matrix_to_list(data['box'])
                 # ##### system.update_box(length)
-                order = order_function.calculate(system)
+                order = self.calculate_order(system)
```

Comment:
I get an error here when running two_sate_dihedral example, as it tries to use read_gromos96_file on a .trr. Do you think the following would work?
```suggestion
                order = self.calculate_order(system, xyz = xyz, vel = vel, box = box)
```
as in cp2k.py and lammps.py?

Reply:
Can you check if the changes in b29cd0a fixes this?